services/ui-src/src/routes/Admin/Admin.tsx

```diff
@@ -134,6 +151,11 @@ const sx = {
   currentBannerFlex: {
     flexDirection: "column",
   },
+  buttonLoading: {
+    width: "13.3rem",
```

Comment:
I'm cool if this is the best way to do this, but couldn't the width just go on the `deleteBannerButton` properties permanently?

Reply:
absolutely, i only did this because I always try to isolate what I'm working on as much as possible from anyone else code, as to not create new probs